macaroons/store.go

```diff
@@ -25,13 +27,25 @@ var (
 	// TODO(aakselrod): Add support for key rotation.
 	defaultRootKeyID = []byte("0")
 
-	// macaroonBucketName is the name of the macaroon store bucket.
-	macaroonBucketName = []byte("macaroons")
+	// encryptedKeyID is the ID of the encryption key, encrypted with a
```

Comment:
ID of -> name of the key that stores

?

Reply:
Comment updated to clarify and fixed to specify that the encrypted key material is everything after the first 32 bytes, since it includes metadata.